# Digital-Gold — a quantitative teardown 🔬
### Correlations and their drift · crash-month behaviour · the small-sleeve overlay

![Signal: None](https://img.shields.io/badge/Signal-None-c0392b?style=flat-square)
![Tradability: Fragile](https://img.shields.io/badge/Tradability-Fragile-dab617?style=flat-square)
![Bitcoin is digital gold?: Not supported](https://img.shields.io/badge/Bitcoin_is_digital_gold%3F-Not_supported-8b949e?style=flat-square)

The deep companion to the [notebook for the curious](01_for_the_curious.ipynb). The 'digital gold' thesis, measured — a high-beta risk asset wearing a safe-haven costume.

> ⚠️ **Not investment advice.** BTC-USD / SPY / GLD daily total returns, 2014–2026 (Yahoo). Sources in [`docs/references.md`](../docs/references.md).

In [1]:
import sys, os
sys.path.insert(0, os.path.abspath("../../.."))  # repo root (quantlab/)
sys.path.insert(0, os.path.abspath(".."))        # study package (digital_gold/)
%matplotlib inline
import matplotlib.pyplot as plt
plt.rcParams["figure.figsize"] = (10, 5.5); plt.rcParams["axes.grid"] = True
import numpy as np, pandas as pd
from digital_gold import data, strategy as st
ret = data.fetch_panel()                       # cache-first (BTC/SPY/GLD pull)
corr = st.correlations(ret)
monthly = (1 + ret).resample("ME").prod() - 1
cb = st.crisis_behavior(monthly)


## Verdict, up front

| Axis | Stamp | Why |
|---|---|---|
| Signal (digital gold) | **None** | corr w/ gold +0.09; falls in crashes |
| Tradability (sleeve) | **Fragile** | 5% BTC Sharpe 0.82→0.95, on −83% DD |
| Is digital gold? | **Not supported** | rising stock corr, no haven |

> 💡 *In plain words:* a return bet dressed as insurance.

## 1 · The claim, steelmanned

- **H₁:** BTC behaves like gold (correlated with gold, uncorrelated with stocks).
- **H₂:** BTC is a safe haven (holds/rises when equities crash).
- **H₃:** it's a store of value (stable enough to be insurance, not a leveraged bet).

## 2 · So what? — what rides on each

H₁/H₂/H₃ are the 'digital gold' case. If they fail, BTC is a risk asset, and any portfolio role is a *return* decision (with tail risk), not a hedging one — the difference between sizing it as insurance vs as a speculative sleeve.

## 3 · How we'd know — the protocol

Correlations (BTC–gold, BTC–stock, first vs second half) → crash-month returns vs gold and stocks → a small-sleeve overlay to see *why* it helped (return, not hedging).

## 4 · The teardown

### 4.1 Correlations and their drift — H₁

In [2]:
print({k:round(v,3) for k,v in corr.items()})

{'btc_stock': 0.225, 'btc_gold': 0.086, 'stock_gold': 0.061, 'btc_stock_first': 0.127, 'btc_stock_second': 0.361}


> 💡 *In plain words:* BTC–gold +0.09 (not gold-like), BTC–stock rising +0.13→+0.36 (increasingly a risk asset). **H₁ rejected.**

### 4.2 Crash-month behaviour — H₂

In [3]:
print({k:(round(v,3) if isinstance(v,float) else v) for k,v in cb.items()})

{'btc_in_crash': -0.177, 'gold_in_crash': -0.004, 'eq_in_crash': -0.095, 'btc_up_share': 0.0, 'gold_up_share': 0.2, 'n_crash': 5}


> 💡 *In plain words:* in equity-crash months BTC −17.7% vs stocks −10.1%, up in **0%** of them; gold held flat. BTC falls *harder* than the thing it's meant to hedge. **H₂ rejected.**

### 4.3 The sleeve — why it still 'helped' (and H₃)

In [4]:
sl = st.sleeve_effect(ret['SPY'], ret['BTC-USD'])
show = sl.copy()
for c in ['ann','vol','max_drawdown']: show[c] = (show[c]*100).round(1)
show['sharpe'] = show['sharpe'].round(2)
display(show[['ann','vol','sharpe','max_drawdown']])

,ann,vol,sharpe,max_drawdown
0% BTC,14.4,17.6,0.82,-33.7
5% BTC,16.9,17.7,0.95,-33.5
10% BTC,19.4,18.5,1.05,-33.5


A 5–10% sleeve raised Sharpe (0.82→0.95→1.05) — but read the source: BTC returned +64%/yr. The 'improvement' is a **return** contribution riding a historic trend, with an −83% asset drawdown underneath. That's not stability; **H₃ rejected** — it's a leveraged growth bet, not a store of value.

## 5 · The verdict

H₁/H₂/H₃ all rejected → Signal `NONE` (as digital gold), Tradability `FRAGILE` (a return sleeve, not a hedge), is-digital-gold `NOT SUPPORTED`. The fingerprinted run is in [`docs/results.md`](../docs/results.md).

## 6 · Could you trade it?

As a small, vol-aware *growth* sleeve sized for catastrophic drawdown — yes; as a haven or inflation hedge — no. The crash-month evidence (Conlon-McGee 2020 on COVID; our 2014–2026 sample) is unambiguous: BTC is risk-on, not insurance.

## 7 · Going further

Forks: (a) vol-target BTC sizing; (b) the post-spot-ETF (2024) correlation regime; (c) BTC vs gold across named crises; (d) BTC's beta to the Nasdaq / liquidity. Backlog: [`docs/pwb_strategies_inventory.md`](../../../docs/pwb_strategies_inventory.md).